package-lock.json

```diff
@@ -3065,172 +2829,173 @@
             "integrity": "sha512-HJSUJmni4BeDHhfzn6nF0sVmd1SMezP7/4F0Lq+aXzmp2xm9O7WXrUtHW/CHlYVtZUbByEvWidHqRtcJXGF2Ng==",
             "peer": true
         },
+        "@types/web-bluetooth": {
```

Comment:
Bluetooth?

Reply:
Is a dependency of one npm module, idk for what purpose, but it's needed